mmdet/integration/nncf/utils.py

```diff
@@ -7,6 +7,13 @@
     _is_nncf_enabled = True
 except ImportError:
     _is_nncf_enabled = False
+except RuntimeError as _e:
+    _is_nncf_enabled = False
+    print('Attention: RuntimeError happened when tried to import nncf')
```

Comment:
maybe it is better to use logging.warning?

Reply:
and why is exception raised here, why do you think that it is absence of gpus?

Reply:
> and why is exception raised here, why do you think that it is absence of gpus?

This is a known issue in NNCF. It can be used for CPU-only training, but it should be built with special flags for that.
Without this fix tests on CPU-only training will fail in OTE, since mmdetection tries to make import NNCF and crashes.